applications/forms.py

```diff
@@ -55,3 +55,78 @@ class Meta:
             "training_passed_at",
         ]
         model = ResearcherRegistration
+
+
+class ResearcherRegistrationSubmissionForm(ResearcherRegistrationPageForm):
+    """
+    Submission form for Researchers
+
+    When submitting an application we want to do extra validation of the
+    fields, however researchers are separate to pages so we take the form used
+    to create/edit researchers and add cross-field validation to it.
+    """
+
+    def clean_does_researcher_need_server_access(self):
+        """
+        Make does_researcher_need_server_access a required field on this form.
+        """
+        does_researcher_need_server_access = self.cleaned_data.get(
+            "does_researcher_need_server_access", None
+        )
+        if does_researcher_need_server_access is None:
+            raise forms.ValidationError("")
```

Comment:
> is this type of ValidationError a surprise for you to find?

Slightly surprising.  Because it's not something we typically see, it's not immediately clear whether this is deliberate, or whether you're planning to go back and add a message later.

If you change it from `raise ValidationError("")` to `ValidationError(self.error_messages["required"], code="required")`, the behaviour will match what [normally happens](https://github.com/django/django/blob/cb8d7ca0ba0adc4fcefc069eae26f842a15755a1/django/forms/fields.py#L126-L128) when validation fails because a required field is missing.

Reply:
An alternative way to make this field required would be to do so in an  `__init__` method, like [this](https://stackoverflow.com/a/24045492).